.github/workflows/kind_e2e_tests.yaml

```diff
@@ -55,11 +55,11 @@ jobs:
           sudo mv yq_linux_amd64.tar.gz /usr/bin
       - name: Parse image version
         id: parse_image_tag
-        run: echo "::set-output name=image_tag::$(yq eval '.images[0].newTag' config/deployments/default/kustomization.yaml)"
+        run: echo "image_tag=$(yq eval '.images[0].newTag' config/deployments/default/kustomization.yaml)" >> $GITHUB_OUTPUT
       - name: Set up Docker buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
       - name: Cache Docker layers
-        uses: actions/cache@v2
+        uses: actions/cache@v3
```

Comment:
There's no need to cache docker layers here as the buildx operation has the same caching (line 78/79)